app/views/posts/index.html.erb

```diff
@@ -6,7 +6,7 @@
   <% @all_posts.each do |post| %>
     <%= render partial: "shared/post", locals: {post: post} %>
```

Comment:
- [ ] <b>REQUIRED:</b> According to afformentioned issues, Now here your app is throwing `@all_posts is undefined` error.

Reply:
✅ Resolved.